src/view/search_view_components/QueryBar.java

```diff
@@ -33,7 +33,7 @@ public QueryBar(SearchViewModel searchViewModel, SwitchViewController switchView
         this.switchResultsPanelController = switchResultsPanelController;
         this.queryAllController = queryAllController;
         this.queryOneController = queryOneController;
-        switchViewButton = new ImageButton(SearchViewModel.BACK_BUTTON_IMAGE_PATH, SearchViewModel.BACK_BUTTON_IMAGE_SCALE);
+        switchViewButton = new JButton("ᐊ"); // or ❮
         switchViewButton.setBorder(null);
```

Comment:
One line of my code was deleted by mistake.

Reply:
I solved the problem and I apologize.